taxi/admin.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Car, Driver, Manufacturer
+
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    search_fields = ["model"]
+    list_filter = ["manufacturer"]
+
+
+@admin.register(Driver)
+class DriverAdmin(UserAdmin):
+    list_display = ["license_number"]
+    fieldsets = UserAdmin.fieldsets + (
+        (("Additional info", {"fields": ("license_number",)}),)
+    )
+    add_fieldsets = UserAdmin.add_fieldsets + (
+        (
+            (
+                "Additional info",
+                {
+                    "fields": (
+                        "first_name",
+                        "last_name",
+                        "license_number",
+                    )
+                },
+            ),
+        )
+    )
+
+
+admin.site.register(Manufacturer)
```

Comment:
Use either a functional approach or utilize decorators to register your models in the `admin.py`.